src/Pos/Generator/Block/Mode.hs

```diff
@@ -182,7 +183,7 @@ mkBlockGenContext bgcParams@BlockGenParams{..} = do
         return BlockGenContext {..}
   where
     -- Genesis utxo is needed only for boot era stakeholders
-    bgcGenesisUtxo =
+    bgcGenesisTxpContext =
```

Comment:
We both know that it's bullshit, can you remove it? `GenesisStakeholders` should come from `MonadBlockGen` instead.